### ISSUE TITLE
Time should be double not float

### ISSUE THREAD

**Issue by meshula** [ddiakopoulos/LabSound#40]
cf. https://github.com/WebAudio/web-audio-api/issues/465

WebAudio is punting on it but float as time in general is bad because of rounding after tens of minutes. For performance on non-double native platforms perhaps internally some integer scheme should be used, instead of wiring doubles all the way through. Perhaps there's another solution...

**Comment by ddiakopoulos** [comment]
Hmmm... but do we *want* to target things that don't have a 64 bit type?

**Comment by meshula** [comment]
Good point. The weird platforms I'm interested in are Tango, and RaspPi 2, and both have double support. For some reason I though the RaspPi wouldn't do it, but it seems fine.

**Comment by ddiakopoulos** [comment]
I had a go at this today and aborted mission. AudioParam uses a summing bus which only operates on floats and from there it's just turtles. I did a quick POC templating things like AudioChannel on float/double, but it got ugly very quickly.

**Comment by meshula** [comment]
Turtles :) I can totally imagine what you found.
